lib/suse/connect/templates/extension_item.text.erb

```diff
@@ -1,11 +1,8 @@
 <%= indent(level) %>\e[1m<%= extension[:name] %>\e[0m<%
-  if extension[:installed]
-    %> \e[32m(Installed)\e[0m
-<%= indent(level) %>Deactivate with: <%= @binary %> \e[31m-d\e[0m -p <%= extension[:activation_code] %><%
-  else %><%
-    if extension[:activated]
-      %> \e[33m(Activated)\e[0m<%
-    end %>
-<%= indent(level) %>Activate with: <%= @binary %><%= '   ' if any_installed %> -p <%= extension[:activation_code] %><% unless extension[:free] %> -r \e[32m\e[1mADDITIONAL REGCODE\e[0m<%
-    end %><%
-  end %>
+unless extension[:available] %> \e[31m(Not available)<% end -%>
+<% if extension[:activated] %> \e[33m(Activated)\e[0m<% end -%>
+<% if extension[:activated] %>
```

Comment:
Is it necessary to have the same conditional twice?

Reply:
The problem is with erb it's terrible hard to get the line breaks right. I tought in this case it's easier to add to conditions instead of messing with the line breaks but if you think it's too much I can fiddle this out.

Reply:
No, it's necessarily hurting anything. Rather keep it if it messes up the newlines.